app/controllers/publishers/vacancies_controller.rb

```diff
@@ -7,6 +7,8 @@ class Publishers::VacanciesController < Publishers::Vacancies::BaseController
 
   helper_method :vacancy_statistics_form
 
+  def start; end
```

Comment:
I think that we should stick to `new` as it's more in line with rails controller convention and I when we reach that action we have not yet created a vacancy so it a good place for that kind a page to be.

Reply:
had a chat offline abouth this and we agreed to leave as is. With the current codebase structure either ways have there own drawbacks. This can be reviews with the future work about forms.